src/core_modules/capture-core/components/DataEntry/withCancelButton.js

```diff
@@ -21,10 +21,6 @@ const getCancelButton = (InnerComponent: React.ComponentType<any>, optionsFn?: ?
 
         getWrappedInstance = () => this.innerInstance;
 
-        getCancelButtonInstance = (cancelButtonInstance: ?any) => {
-            this.props.cancelButtonRef && this.props.cancelButtonRef(cancelButtonInstance);
-        }
-
```

Comment:
I might not understanding the whole spectrum of how this is used. It looked to me it wasnt in use.